main-glossary.md

```diff
@@ -0,0 +1,16 @@
+---
+description: A list of words and terms relating to the Angle Protocol
+---
+
+# 📚 Glossary - Angle Protocol
+
+Here, you will find a list of the terms and expressions that are used across the protocol.
+
+If you still have some questions, do not hesitate to reach out on the [Angle Discord](https://discord.gg/67WSSZqBG6) 🕹️.
+
+| Term | Description |
+| ---- |-----------  |
+| Stablecoin | A cryptocurrency, or token, that is pegged to another asset. This means that the stablecoin in question is designed to mirror the value of the asset it is pegged to.        |
+| agToken | Angle stablecoins. agTokens are stablecoins issued by the Angle Protocol modules. They are named from the asset they are pegged to. For example, agEUR is pegged to the EUR.   |
+| Module | A set of smart contracts interacting together for a specific use case.          |
+| Collateral | Any token that can be used to back some type of loan. In Angle, tokens are used as collateral to borrow agTokens.   |
```

Comment:
Core Module, Borrowing Module, Governance, ANGLE, veANGLE, TVL and so on are things we can define here for newbies